peps/pep-0759.rst

```diff
@@ -0,0 +1,472 @@
+
+   bringing about more people to utilize PyPI's repository features, an
+   altogether good thing given the global CDN powering PyPI providing speed
+   ups for a lot of people[...]
+
+Instead of external hosting, PEP 470 promoted the use of explicit multiple
+repositories, providing full package indexing and artifact hosting, and
+enabled through installer tool support, such as ``pip install
+--extra-index-url`` allowing ``pip`` to essentially treat multiple
+repositories as `one single global repository
+<https://pip.pypa.io/en/stable/cli/pip_install/#cmdoption-extra-index-url>`__
+for package installation resolution. Because this has been the blessed norm
+for so many years, all Python package installation tools support querying
+multiple indexes for dependency resolution.
+
+The problem with multiple indexes
+=================================
+
+Why then does this PEP propose to allow a more limited form of external
+hosting, and how does this proposal avoid the problems documented in PEP 470?
+
+One well-known problem that consolidating multiple indexes enables is
+`dependency confusion attacks
+<https://medium.com/@alex.birsan/dependency-confusion-4a5d60fec610>`__, to
+which Python *can* be particularly vulnerable, due to the algorithm that ``pip
+install`` uses for resolving package dependencies and preferred versions. The
+``uv`` tool addresses this by supporting an additional `index strategy
+<https://docs.astral.sh/uv/reference/settings/#index-strategy>`__ option,
+whereby users can select between, e.g. a ``pip``-compatible strategy, and a
+more limited strategy that prevents such dependency confusion attacks.
+
+:pep:`708` provides additional background about dependency confusion attacks,
+and takes a different approach to preventing them. At its core, PEP 708 allows
+repository owners to indicate that projects track across different
+repositories, which allows installers to determine how to treat the global
+package namespace when combined across multiple repositories. PEP 708 has been
+provisionally accepted, pending several required conditions as outlined in PEP
+708, some of which may have an indeterminate future. As PEP 708 itself says,
+this won't by itself solve dependency confusion attacks, but is one way to
+provide enough information to installers to help minimize these attacks.
+
+While there can still be valid use cases for standing up a totally independent
+package index (such as providing richer platform support for GPUs until a
+fully formed `variant proposal
+<https://discuss.python.org/t/selecting-variant-wheels-according-to-a-semi-static-specification/53446>`__
+is accepted), this PEP takes a different, simpler approach and doesn't replace
+any of the existing, proposed, or approved package index cooperation
+specifications.
+
+This PEP also preserves the core purpose of PyPI, and allows it to
+remain the traditional, canonical, centralized index of all Python
+packages.
+
+It's not just about indexes
```

Comment:
```suggestion
It's more than indexes
```

Reply:
Thanks @willingc - I actually struggled a bit with this heading while I was writing the PEP, but I have a different approach that I think will help clarify this .